chain/network/src/peer/peer_actor.rs

```diff
@@ -485,18 +512,6 @@ impl PeerActor {
             }
         }
 
-        // Verify that the received partial edge is valid.
```

Comment:
where is the verification now?

Reply:
the edge is verified within network_state.register() call which calls add_edge().